ghost/i18n/locales/ru/ghost.json

```diff
@@ -62,7 +62,7 @@
     "View in browser": "Просмотреть в браузере",
     "Welcome back to {siteTitle}!": "Добро пожаловать на {siteTitle}, с возвращением!",
     "Welcome back to {siteTitle}! Your verification code is {otc}.": "",
-    "Welcome back! Here's your code to sign in to {siteTitle}. For your security, it's only valid for 24 hours": "С возвращением! Вот ваш код для входа на {siteTitle}. В целях безопасности он действителен только в течение 24 часов",
+    "Welcome back! Here's your code to sign in to {siteTitle}": "",
```

Comment:
We can probably keep the existing translation for this one, minus the last sentence.